python/GafferUI/UIEditor.py

```diff
@@ -111,6 +111,12 @@ def __init__( self, scriptNode, **kw ) :
 						MetadataWidget.FileSystemPathMetadataWidget( key = "icon" )
 					)
 
+					_Label( "Scale" )._qtWidget().setFixedWidth( 40 )
```

Comment:
Small thing, but the layout feels better to me if this is just `GafferUI.Label( "Scale" )`, which removes the gap. I wonder if the same is also better for the "Top/Bottom/Left/Right" labels below.

Reply:
Sounds good to me, I've switched over to regular `GafferUI.Label()` in 7a0274e.

Tried the same for "Top/Bottom/Left/Right", which I don't mind after adding a bit of extra spacing to the ListContainer. See what you think with c58a54e89af5088ceffb579ffaa76579e7880b23.